src/driver/mysql/MysqlQueryRunner.ts

```diff
@@ -1296,6 +1296,7 @@ export class MysqlQueryRunner extends BaseQueryRunner implements QueryRunner {
                     tableColumn.name = dbColumn["COLUMN_NAME"];
                     tableColumn.type = dbColumn["DATA_TYPE"].toLowerCase();
 
+                    tableColumn.unsigned = tableColumn.zerofill ? true : dbColumn["COLUMN_TYPE"].indexOf("unsigned") !== -1;
```

Comment:
`tableColumn.zerofill` is set below, why did you move it?

Reply:
Good catch. The value of `tableColumn.unsigned` is used inside the `isDefaultColumnWidth()` method, so it needs to be set prior to checking the width, which is why I moved it up. I missed the dependency on the `zerofill` value. I'll move the `zerofill` assignment above the `unsigned` assignment.